src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = sourceString
+    .split(';')
+    .filter((style) => style.includes(':'))
```

Comment:
The 'filter' method is used correctly to exclude any string that does not include a colon, which would not be a valid CSS declaration.